information-schema/information-schema-tidb-index-usage.md

```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_INDEX_USAGE
+summary: 了解 information_schema 表 `TIDB_INDEX_USAGE`。
+---
+
+# TIDB_INDEX_USAGE
+
+`TIDB_INDEX_USAGE` 记录了当前节点所有 Index 的访问统计信息。
+
+{{< copyable "sql" >}}
+
```

Comment:
```suggestion
```
This copy syntax is no longer needed for the current docs website. Please remove it from this doc.